src/subcommand/server.rs

```diff
@@ -144,8 +144,12 @@ impl Server {
 
       let router = Router::new()
         .route("/", get(Self::home))
-        .route("/block-count", get(Self::block_count))
         .route("/block/:query", get(Self::block))
+        .route("/blockcount", get(Self::block_count))
```

Comment:
It's kinda difficult to remove `/blockcount` because we rely on it in our tests quite heavily